docs/integrations/code-conventions.md

```diff
@@ -600,6 +600,34 @@ You may also use ```headerTransform``` to convert the existing keys into formatt
  
  <img width="758" src="../doc_imgs/integrations/50575199-fd5d0a00-0e01-11e9-9d54-944eb7c6f287.png"></img>
 
+### IndicatorsTimeline
+The `IndicatorTimeline` is an optional object (available from Server version 5.5.0 and up) . It is only applicable for commands that operate on indicators. It is a dictionary (or list of dictionaries) of the following format:
+When `IndicatorTimeline` data is returned in an entry, the timeline section of the indicator whose value was noted in the timeline data will be updated (and is viewable in the indicator's view page in Cortex XSOAR as can be seen in the attached image).
+
+<img width="758" src="../doc_imgs/integrations/timeline_section.png"></img>
+
+**What value should be used for the `'Category'` field of a `timeline` data object?**  
+Any Cortex XSOAR integration command that returns `timeline` data should include the `'Category'` value of `'Integration Update'`. When returning `timeline` data from a Cortex XSOAR automation, the value passed to the `'Category'` field should be `'Automation Update'`.
+
+**So when should one include a timeline object in an entry returned to the war room?**  
+The answer is any time that a command operates on an indicator. A good indicator (pun intended?) of when `timeline` data should be included in an entry is to look and see if the command returns a `DBotScore` or entities as described in our [context standards documentation](../integrations/context-standards) to the entry context. A common case is reputation commands, i.e. `!ip`, `!url`, `!file`, etc. When implementing these commands in integrations, `timeline` data should be included in the returned entry. To see an example of an integration that returns entries with `timeline` data, take a look at our [AbuseIPDB integration](https://github.com/demisto/content/blob/14148b68f5030a64c6fe6f7cf5af4f184e93abad/Packs/AbuseDB/Integrations/AbuseDB/AbuseDB.py#L215).
+
+
+| Arg        | Type   | Description                                                                                                                                                                                |
+|------------|--------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| indicators | list   | Expects a list of indicators, if a dict is passed it will be put into a list.                                                                              |
+| category   | str    | Indicator category.
+| message    | str    | Indicator message.                                                                                                                                         |
+
+**Example**
+```python
+timeline = IndicatorsTimeline(
+      indicators=[args.get('ips')],
+      category='Some category',
```

Comment:
no need to provide category in the example if category will have defaults

Reply:
I will add another example without the defaults.